index.html

```diff
@@ -11,50 +11,300 @@
-        <p class="intro-info">
-          I’m a software developer! I can help you build a product , feature or
-          website Look through some of my work and experience! If you like what
-          you see and have a project you need coded, don’t hestiate to contact
-          me.
-        </p>
-        <p class="connect">LET’S CONNECT</p>
-        <ul class="social-icons">
-          <li class="list">
-            <a href="#">
-              <img src="assets/images/icon1.svg" alt="icon1" />
-            </a>
-          </li>
-          <li class="list">
-            <a href="#">
-              <img src="assets/images/Linkedin icon.svg" alt="icon2" />
-            </a>
-          </li>
-          <li class="list">
-            <a href="#">
-              <img src="assets/images/icon2.svg" alt="icon3" />
-            </a>
-          </li>
-          <li class="list">
-            <a href="#">
-              <img src="assets/images/icon3.svg" alt="icon4" />
-            </a>
-          </li>
-        </ul>
+    <div class="container">
+      <!-- Hearder and headline section -->
+      <header class="header">
+        <a class="logo" href="#">My Logo</a>
+        <nav>
+          <img src="assets/images/Union.jpg" alt="nav-icon" class="navbar" />
+        </nav>
+      </header>
+      <section class="headline">
+        <div class="introduction">
+          <h1 class="heading">I’m Mayito Glad to see you!</h1>
+          <p class="intro-info">
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </p>
+          <p class="connect">LET’S CONNECT</p>
+          <ul class="social-icons">
+            <li class="list">
+              <a href="#">
+                <img src="assets/images/icon1.svg" alt="icon1" />
+              </a>
+            </li>
+            <li class="list">
+              <a href="#">
+                <img src="assets/images/Linkedin icon.svg" alt="icon2" />
+              </a>
+            </li>
+            <li class="list">
+              <a href="#">
+                <img src="assets/images/icon2.svg" alt="icon3" />
+              </a>
+            </li>
+            <li class="list">
+              <a href="#">
+                <img src="assets/images/github.svg" alt="icon4" />
+              </a>
+            </li>
+            <li class="list">
+              <a href="#">
+                <img src="assets/images/thumbs.svg" alt="icon4" />
+              </a>
+            </li>
+          </ul>
+        </div>
+      </section>
+
+      <!-- works section -->
+      <div class="works">
+        <section class="projects">
+          <img
+            src="assets/images/project-1.svg"
+            class="project-1"
+            alt="project-one"
+          />
```

Comment:
Your project looks great. I think that it would be more semantic to have the container of the works section use a `section` tag. What do you think?